pkg/orchestrator/openshift/openshift.go

```diff
@@ -289,14 +249,9 @@ func (c *openshiftContext) OpenshiftAfterCompleteDrainNode(ctx context.Context,
 	return true, nil
 }
 
-func (c *openshiftContext) GetNodeMachinePoolName(ctx context.Context, node *corev1.Node) (string, error) {
-	// if it's not an openshift cluster we return error
-	if !c.IsOpenshiftCluster() {
-		return "", fmt.Errorf("not an openshift cluster")
-	}
-
-	// hyperShift cluster don't have machine config
-	if c.IsHypershift() {
+func (c *OpenshiftContext) GetNodeMachinePoolName(ctx context.Context, node *corev1.Node) (string, error) {
```

Comment:
can you change all methods that are not related to the interface implementation to be private ?

Reply:
ack